modules/ossm-about-cluster-wide-migration.adoc

```diff
@@ -0,0 +1,59 @@
+
+== Control planes and root certificate
+
+During the migration, the control plane associated with the 2.6 installation and the control plane associated with the 3.0 installation must share the same root certificate. This can be achieved by installing the 3.0 control plane to the same namespace as 2.6 control plane. The migration procedures show how to verify the root certificate is shared.
+
+== Control planes, discovery selectors, and namespace access
+
+Both control planes must have access to all namespaces in the mesh. During the migration, some proxies will be controlled by the 3.0 control plane while other proxies will remain controlled by the 2.6 control plane. To ensure that mesh communication works during the migration, both control planes must be aware of the same set of services. 
+
+Discovery selectors specify the set of namespaces that comprise the mesh. If you are using discovery selectors, you must ensure that the `discoverySelectors` label defined in the {SMProduct} 3.0 {istio} resource match the namespaces that comprise the {SMProduct} 2.6 mesh. 
+
+If the 2.6 control plane uses discovery selectors that were automatically generated by the control plane, such as `maistra.io/member-of` label, the labels are removed during the migration process. To use discovery selectors in the 3.0 installation you must apply new labels to the application namespaces. See "Scoping the service mesh with DiscoverySelectors" for more information.
+
+== Control planes and network policies
+
+During the migration, ensure that no network policies block the traffic between the `istiod` resource and the proxies for the two control planes. By default, {SMProduct} 2.6 manages network policies that block traffic to the 3.0 control plane. In the pre-migration checklist, you are instructed to disable the network policies. However, you can choose to manually re-create the network policies. The newly created network policies must allow traffic for both control planes. When a dataplane namespace is migrated to 3.0, the `maistra.io/member-of` label automatically gets removed. Use a different label in the network policies. See "Migration of Network Policies" for more information.
+
+[NOTE]
+====
+If network policies are configured incorrectly they can disrupt mesh traffic. When performing the migration, exercise caution when creating network policies to prevent traffic disruption. See "Network Policy migration" for more information.
+====
+
+== Control planes and sidecar injection
+
+During the migration, only one control plane will perform sidecar injection. This is achieved by correctly using injection labels. See "Installing the Sidecar" for more information.
+
+[NOTE]
+====
+During the migration, you must disable 2.6 injector. In this documentation, examples use the `maistra.io/ignore-namespace: "true"` label to prevent the 2.6 control plane from injecting a proxy in the namespace, but your installation might use a different label.
```

Comment:
```suggestion
During the migration, you must disable 2.6 injector. In this documentation, examples use the `maistra.io/ignore-namespace: "true"` label to prevent the 2.6 control plane from injecting a proxy in the namespace.
```

Reply:
It's actually required to use this label.